src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/AuditEvent.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.AuditEventType
+import java.time.LocalDateTime
+
+@Schema(description = "Describes an audit event request")
+data class AuditEvent(
+  @Schema(description = "The internal ID of the audit event", example = "1234")
+  val id: Long? = null,
+
+  @Schema(description = "The internal ID of the licence that this event related to, or null if unrelated to a licence", example = "1234")
+  val licenceId: Long? = null,
+
+  @Schema(description = "The date and time of the event", example = "12/01/2022 23:14:23")
+  @JsonFormat(pattern = "dd/MM/yyyy HH:mm:ss")
+  val eventTime: LocalDateTime = LocalDateTime.now(),
+
+  @Schema(description = "Username who initiated the event, if a user event, or SYSTEM if an automated event", example = "X63533")
+  val username: String? = null,
```

Comment:
Could it default to SYSTEM if username is not supplied?

Reply:
Yes, will add.